dispatcher_test.go

```diff
@@ -28,28 +28,33 @@ import (
 	. "go.uber.org/yarpc"
 	"go.uber.org/yarpc/api/transport"
 	"go.uber.org/yarpc/api/transport/transporttest"
+	"go.uber.org/yarpc/internal/observerware"
 	"go.uber.org/yarpc/transport/http"
 	"go.uber.org/yarpc/transport/tchannel"
 
 	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"github.com/uber-go/tally"
 	"go.uber.org/zap"
 )
 
-func basicDispatcher(t *testing.T) *Dispatcher {
+func basicConfig(t testing.TB) Config {
```

Comment:
Why not? Keeps all the helpers usable in tests and benchmarks, in case that comes up later.